hardhat.config.js

```diff
@@ -57,20 +57,58 @@ for (const f of fs.readdirSync(path.join(__dirname, 'hardhat'))) {
 const withOptimizations =
   argv.enableGasReport || argv.compileMode === 'production';
 
+const requestFaucet = async (wallet, amount) => {
+  // TODO: create a wrapper
+  if (process.env.USE_FAUCET === 'true') {
+    console.log('Skipping faucet request');
+    return;
+  }
+  console.log('Requesting faucet...');
+  const data = { amount: amount, wallet: wallet };
+  const faucetUrl = process.env.FAUCET_URL;
```

Comment:
and here too, why process.env.FAUCET_URL instead of Config.faucetUrl like below?

Reply:
Config.faucetUrl is a typo, no typescript is allowed here

Reply:
fixed (Config.faucetUrl has been removed from the config)